Hosts/Console/TddStud10App/TddStud10App.csproj

```diff
@@ -180,6 +189,7 @@
       <Generator>ResXFileCodeGenerator</Generator>
       <LastGenOutput>Resources.Designer.cs</LastGenOutput>
     </EmbeddedResource>
+    <None Include="TddStud10AppCodeAnalysisRules.ruleset" />

```

Comment:
shouldnt we merge all the rulesets into 1 file at the top level and use that for all the projects?